docs/settings.rst

```diff
@@ -141,6 +141,25 @@ Language Detection
     datetime.datetime(2015, 12, 4, 0, 0)
 
 
+Default Languages
++++++++++++++++++
+
+``DEFAULT_LANGUAGES``: It is a ``list`` of language codes in ISO 639 that will be used as default 
+languages for parsing when language detection fails. eg. ["en", "fr"]:
+
+    >>> from dateparser import parse
+    >>> parse('3 de marzo de 2020', settings={'DEFAULT_LANGUAGES': ["es"]})
+
+.. note:: When using this setting, these languages will be tried after trying with the detected languages with no success. It is especially useful when using the ``detect_languages_function`.
+Optional language detect
```

Comment:
Probably a blank line is needed between these two lines. Also the title could be adjusted to "Optional language detection" perhaps?

Reply:
Updated thanks 👍🏻